molo/core/tests/test_models.py

```diff
@@ -177,6 +177,10 @@ def test_get_parent_section_for_article(self):
         parent = article.get_parent_section()
         self.assertEqual(parent.pk, self.yourmind_sub.pk)
 
+    def test_get_parent_section_for_section(self):
+        parent = self.yourmind_sub.get_parent_section()
+        self.assertEqual(parent.pk, self.yourmind.pk)
```

Comment:
Is this test covering all the locale options that this method needs to deal with? or are the others tested somewhere else?

Reply:
It is included in the tests that are failing